From aca79c027483f86e85043f403321a8d8016ca007 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Sun, 3 Jul 2022 16:44:39 +0200
Subject: [PATCH] keyboard: Fix updating suggestions visibility

Commit 237ba24dbeae added corresponding API to the keyboard actor,
but ended up calling it on the KeyboardManager.

Add the expected method to KeyboardManager that forwards the request
to the keyboard.

https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/5630

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2359>
(cherry picked from commit eff23a87c36a6a96c9abab09ab27a4bb35ab1b1f)
---
 js/ui/keyboard.js | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/js/ui/keyboard.js b/js/ui/keyboard.js
index 9e5dbffbb1..e13652bdfe 100644
--- a/js/ui/keyboard.js
+++ b/js/ui/keyboard.js
@@ -1273,6 +1273,10 @@ var KeyboardManager = class KeyBoardManager {
             this._keyboard.resetSuggestions();
     }
 
+    setSuggestionsVisible(visible) {
+        this._keyboard?.setSuggestionsVisible(visible);
+    }
+
     maybeHandleEvent(event) {
         if (!this._keyboard)
             return false;
-- 
GitLab

